Resources/Prototypes/_Scp/Maps/scp_complex_compact.yml

```
@@ -1,16 +1,17 @@
 - type: gameMap
-  id: ScpComplexMini
-  mapName: 'Мини-комплекс SCP'
-  mapPath: /Maps/_Scp/scp_complex_mini.yml
+  id: ScpComplexCompact
+  mapName: 'Комплекс Типа - Compact'
+  mapPath: /Maps/_Scp/scp_complex_compact.yml
   maxRandomOffset: 0
   randomRotation: false
   minPlayers: 0
+  maxPlayers: 30
   stations:
-    ScpComplexMini:
+    ScpComplexCompact:
       stationProto: StandardScpStation
       components:
         - type: StationNameSetup
-          mapNameTemplate: 'Зона-{0}'
+          mapNameTemplate: 'Зона-{0} Типа - Compact'
```

```suggestion
          mapNameTemplate: 'Зона-{0} типа - Compact'
```

с маленькой буквы, и так везде